saltgui/index.html

```diff
@@ -200,8 +232,8 @@ <h2 class='time'></h2>
 
       <div class='route' id='page_jobs'>
           <div class='dashboard'>
-            <div id='jobs_page' class='panel jobs-list'>
-              <h1 id="jobs_title">Recent Jobs</h1>
+            <div id='jobs_page' class='panel jobs-list' style="margin-right: 0;">
+              <h1 id="jobs_title">Last 20 Jobs</h1>
```

Comment:
I would still call it `Recent Jobs`. Because there might not be 20 visible, and we are making a selection on the jobs that we show.

Reply:
Will revert this.